src/components/modals/BaseModal.tsx

```diff
@@ -17,7 +17,7 @@ export const BaseModal = ({ title, children, isOpen, handleClose }: Props) => {
         className="fixed inset-0 z-10 overflow-y-auto"
         onClose={handleClose}
       >
-        <div className="flex min-h-full items-center justify-center py-10 px-4 text-center sm:p-0">
+        <div className="w: flex min-h-full items-center justify-center py-10 px-4 text-center sm:p-0">
```

Comment:
what is the `w:` for here?

Reply:
I thought deleted, It kept moving things around after I saved the file, must have missed deleting it.